dashboard/app/controllers/ops/districts_controller.rb

```diff
@@ -48,7 +48,7 @@ def district_params
           params[:district][:contact][:email] != @district.try(:contact).try(:email)
         # adding/changing district contact
         params[:district][:contact_id] =
-          User.find_or_create_district_contact(contact_params(params[:district].delete(:contact)), current_user).id
+          User.find_or_create_teacher(contact_params(params[:district].delete(:contact)), current_user).id
```

Comment:
Is this controller still in use?

Reply:
From the PR description:

When I asked about it, aoby responded:

> Ah, that ops dashboard is deprecated and on slate for demolition; so if that's the only place it's used we could remove the constant and even clear that permission from the DB

Reply:
No. Everything under `ops` is deprecated and on PLC's backlog to remove.

Reply:
Is this work tracked?  I feel like this has been on the backlog for over a year 😛.

Reply:
I can take this if the work is just to delete all the models/controllers/views/tests associated with the `::OPS` namespace?

Reply:
Lol, yes it has been a while. I started a few months ago and got sidetracked, since been slammed with more timely PLC work. It's tracked [here](https://trello.com/c/TP8Sg8es/168-deprecate-old-ops-dashboard-ui-model-data).